packages/react/src/ReactNodeViewRenderer.tsx

```diff
@@ -150,6 +150,10 @@ export class ReactNodeView<
       className: `node-${this.node.type.name} ${className}`.trim(),
     })
 
+    if (this.editor.options.addTypeAttributes) {
+      this.renderer.element.setAttribute('data-node-view', '')
```

Comment:
Maybe `data-node-view-wrapper`, which is more consistent to the usage with `data-node-view-content`?

Reply:
done